code/game/machinery/vending/vending.dm

```diff
@@ -30,49 +31,71 @@ GLOBAL_LIST_EMPTY_TYPED(total_vending_machines, /obj/structure/machinery/vending
+	var/vend_power_usage = 150
 
-	var/active = 1 //No sales pitches if off!
-	var/delay_product_spawn // If set, uses sleep() in product spawn proc (mostly for seeds to retrieve correct names).
-	var/vend_ready = 1 //Are we ready to vend?? Is it time??
-	var/vend_delay = 1 SECONDS //How long does it take to vend?
-	var/datum/data/vending_product/currently_vending = null // A /datum/data/vending_product instance of what we're paying for right now.
+	/// if false, the vendor will not send sales pitches
+	var/active = TRUE
+	/// if set, uses sleep() in product spawn proc (mostly for seeds to retrieve correct names).
+	var/delay_product_spawn
+	/// if the vendor can vend an item
+	var/vend_ready = TRUE
+	/// time delay between items can be vended
+	var/vend_delay = 1 SECONDS
+
+	// A /datum/data/vending_product instance of what we're paying for right now.
+	var/datum/data/vending_product/currently_vending = null
 
 	// To be filled out at compile time
 	var/list/products = list() // For each, use the following pattern:
 	var/list/contraband = list() // list(/type/path = amount,/type/path2 = amount2)
 	var/list/premium = list() // No specified amount = only one in stock
 	var/list/prices  = list() // Prices for each item, list(/type/path = price), items not in the list don't have a price.
 
+	/// string of slogans separated by semicolons, optional
+	var/product_slogans = ""
+	/// string of small ad messages in the vending screen - random chance
+	var/product_ads = ""
+
 	/// Used to increase prices of a specific type of vendor.
 	var/product_type = VENDOR_PRODUCT_TYPE_UNDEF
 
-	var/product_slogans = "" //String of slogans separated by semicolons, optional
-	var/product_ads = "" //String of small ad messages in the vending screen - random chance
 	var/list/product_records = list()
 	var/list/hidden_records = list()
 	var/list/coin_records = list()
 	var/list/slogan_list = list()
-	var/list/small_ads = list() // small ad messages in the vending screen - random chance of popping up whenever you open it
-	var/vend_reply //Thank you for shopping!
+
+	/// sent after vending an item
+	var/vend_reply
 	var/last_reply = 0
-	var/last_slogan = 0 //When did we last pitch?
-	var/slogan_delay = 600 //How long until we can pitch again?
-	var/icon_vend //Icon_state when vending!
-	var/icon_deny //Icon_state when vending!
-	var/seconds_electrified = 0 //Shock customers like an airlock.
-	var/shoot_inventory = FALSE //Fire items at customers! We're broken!
-	var/shut_up = 0 //Stop spouting those godawful pitches!
-	var/extended_inventory = 0 //can we access the hidden inventory?
-	var/panel_open = FALSE //Hacking that vending machine. Gonna get a free candy bar.
+	/// when did we last pitch?
+	var/last_slogan = 0
+	/// how long until we can pitch again?
+	var/slogan_delay = 600
+	/// icon_state when vending
+	var/icon_vend
+	/// icon_state when failing to vend
+	var/icon_deny
+	/// shock customers like an airlock.
+	var/seconds_electrified = 0
```

Comment:
Should probably use timers to deal with electrification time rather than - I'm guessing a process statement